src/Microsoft.Azure.WebJobs.Logging.ApplicationInsights/Extensions/ApplicationInsightsServiceCollectionExtensions.cs

```diff
@@ -340,8 +346,13 @@ internal static LoggerFilterOptions CreateFilterOptions(LoggerFilterOptions regi
                 {
                     quickPulseProcessor = new QuickPulseTelemetryProcessor(next);
```

Comment:
While we're in here... if LiveMetrics is disabled there's also no need to add this Processor. Can we exclude that as well? I suspect there'll be a slight perf bump from that too.

Reply:
Done